examples/structured_2d_dgsem/elixir_shallowwater_three_mound_dam_break.jl

```diff
@@ -0,0 +1,114 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the shallow water equations with a discontinuous
+# bottom topography function
+
+equations = ShallowWaterEquations2D(gravity_constant=9.81, H0=1.4)
+
+"""
+    initial_condition_three_mounds(x, t, equations::ShallowWaterEquations2D)
+
+Initial condition including two dam breaks, one on each end of the x dimension. The bottom
+topography is given by one large and two smaller mounds. Those are flooded by the water for t > 0.
+Periodic boundary conditions were used. To smooth the discontinuities, a logistic function is applied.
```

Comment:
```suggestion
To smooth the discontinuities, a logistic function is applied.
```
You don't need to mention the boundary conditions in the initial conditions docstring.